src/main/java/gameboy/gamepack/racinggame/view/InputView.java

```diff
@@ -1,25 +1,81 @@
 package gameboy.gamepack.racinggame.view;
 
-import java.util.InputMismatchException;
-import java.util.Scanner;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import gameboy.gamepack.racinggame.data.vo.Name;
+import gameboy.gamepack.racinggame.exception.InvalidRacerNameException;
 
 public class InputView {
 
-    public static int inputNumber(String ask) {
+    private static final int MAX_NAME_SIZE = 5;
+    private static final String CARS_NAME_SEPARATOR = ",";
+
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static int inputRaceCount() {
+        return inputNumber("시도할 회수는 몇 회 인가요?");
+    }
+
+    public static List<Name> inputCarsName() {
+        return inputCarsName("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).").stream().map(Name::new).toList();
```

Comment:
모든 사람들이 와이드 모니터를 사용하고 있지 않습니다. 😅
```suggestion
        return inputCarsName("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).")
        .stream()
        .map(Name::new)
        .toList();
```